### DISCUSSION TITLE
Utilizar o Handler para mais de uma entidade?

### DISCUSSION THREAD

**Discussion by mathfrance** [balta-io/forum#297] [Tira Dúvidas] [answered]
Olá estou realizando o curso do TodoApp e adaptando a API criada anteriormente na carreira. Eis que cheguei nesse ponto da utilização dos Handlers e a dúvida é se eu posso criar um Handler Account que lida tanto com a entidade User quanto a Role? Ou é preciso separar os conceitos "Cada entidade com seu Handler".

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @mathfrance, como vai?
Não existe um "jeito certo". Você pode começar com um Handler apenas e se achar que está muito grande, vai dividindo.

**Reply by mathfrance** [reply]
Entendi, obrigado @andrebaltieri.
